server/migrations/20220118165920_created_intial_tables.js

```diff
@@ -1,162 +1,165 @@
-      table.timestamps(true, true);
-    })
-    .createTable("wishlist", (table) => {
-      table.increments("id").primary();
-      table.integer("user_id").unsigned().notNullable();
-      table.foreign("user_id").references("users.id");
-      table.integer("product_id").unsigned().notNullable();
-      table.foreign("product_id").references("products.id");
-      table.timestamps(true, true);
-    })
-    .createTable("reviews", (table) => {
-      table.increments("id").primary();
-      table.integer("rating").notNullable();
-      table.string("comment").notNullable();
-      table.integer("product_id").unsigned().notNullable();
-      table.foreign("product_id").references("products.id");
-      table.integer("user_id").unsigned().notNullable();
-      table.foreign("user_id").references("users.id");
-      table.timestamps(true, true);
-    })
-    .createTable("payment", (table) => {
-      table.increments("id").primary();
-      table.string("type").notNullable();
-      table.string("status").notNullable();
-      table.integer("order_id").unsigned().notNullable();
-      table.foreign("order_id").references("orders.id");
-      table.timestamps(true, true);
-    })
-    .createTable("shipping", (table) => {
-      table.increments("id").primary();
-      table.string("type").notNullable();
-      table.string("status").notNullable();
-      table.integer("order_id").unsigned().notNullable();
-      table.foreign("order_id").references("orders.id");
-      table.timestamps(true, true);
-    })
-    .createTable("featured_products", (table) => {
-      table.increments("id").primary();
-      table.integer("product_id").unsigned().notNullable();
-      table.foreign("product_id").references("products.id");
-      table.timestamps(true, true);
-    })
-    .createTable("variants", (table) => {
-      table.increments("id").primary();
-      table.string("size");
-      table.string("color");
-      table.string("type");
-      table.float("price", 0.0);
-      table.boolean("is_default", false);
-      table.integer("product_id").unsigned().notNullable();
-      table.foreign("product_id").references("products.id");
-      table.timestamps(true, true);
-    })
-    .createTable("variant_images", (table) => {
-      table.increments("id").primary();
-      table.string("image_url").notNullable();
-      table.integer("variant_id").unsigned().notNullable();
-      table.foreign("variant_id").references("variants.id");
-      table.timestamps(true, true);
-    })
-    .createTable("cart_items", (table) => {
-      table.increments("id").primary();
-      table.integer("quantity").notNullable();
-      table.integer("variant_id").unsigned().notNullable();
-      table.foreign("variant_id").references("variants.id");
-      table.integer("cart_id").unsigned().notNullable();
-      table.foreign("cart_id").references("cart.id");
-      table.timestamps(true, true);
-    });
+  return (
```

Comment:
why is this file showing as updated? what did you change here? make sure you don't remove any of the changes that others might have done.

Reply:
no other changes